### DISCUSSION TITLE
Compressor

### DISCUSSION THREAD

**Discussion by JustCauseWhyNot** [wwmm/easyeffects#2466] [Q&A] [answered]
Is it possible to have a headphone eq globally while also having a compressor effect for just certain applications. I find that even if I exclude a program the compressor still makes all programs quite when triggered by a non excluded program. I also don't want to exclude a program from the headphones eq. And I am referring to the input compressor effect not output. Thanks for any answers.

**Comment by wwmm** [comment, marked as answer]
>Is it possible to have a headphone eq globally while also having a compressor effect for just certain applications

No, it isn't. This would require one effects pipeline for each application. But EasyEffects is designed around one pipeline for all. Large sections of our code would have to be rewritten to allow different effects for different audio clients.

**Reply by JustCauseWhyNot** [reply]
Thanks for the answer
